src/deepsparse/server/README.md

```diff
@@ -0,0 +1,165 @@
+## DeepSparse Server 🔌
+
+```bash
+pip install deepsparse[server]
+```
+
+The DeepSparse inference server allows you to serve models and pipelines for deployment in HTTP. The server runs on top of the popular FastAPI web framework and Uvicorn web server. Currently, the server only supports NLP tasks; however, support for computer vision will soon be released in upcoming versions.
+
+ - Run `deepsparse.server --help` to lookup the available CLI arguments.
+
+```
+        --host TEXT                     Bind socket to this host. Use --host 0.0.0.0
+                                        to make the application available on your
+                                        local network. IPv6 addresses are supported,
+                                        for example: --host '::'. Defaults to
+                                        0.0.0.0
+
+        --port INTEGER                  Bind to a socket with this port. Defaults to
+                                        5543.
+
+        --workers INTEGER               Use multiple worker processes. Defaults to
+                                        1.
+
+        --log_level                     Sets the logging level. Defaults to info.
+
+        --config_file TEXT              Configuration file containing info on how to
+                                        serve the desired models.
+
+        --task TEXT                     The task the model_path is serving. For
```

Comment:
Check the companion main.py file as there is no period at the end of the description there, but there is one here!

Reply:
will add periods.